src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationAnalysisChartPresenter.cs

```diff
@@ -158,7 +158,7 @@ private void addObservedDataForOutput(IGrouping<string, OutputMapping> outputMap
          var outputPath = outputMappingsByOutput.Key;
          SelectColorForPath(outputPath);
 
-         outputMappingsByOutput.Each(outputMapping => AddDataRepositoryToEditor(outputMapping.WeightedObservedData));
+         AddDataRepositoriesToEditor(outputMappingsByOutput.Select(x=>x.WeightedObservedData.ObservedData));
```

Comment:
Interesting is that we always have multiple and we are adding one at a time. Now we don't have to use an Each